### DISCUSSION TITLE
Context Variables for printing reports

### DISCUSSION THREAD

**Discussion by lookme2** [inventree/InvenTree#2754] [Q&A] [answered]
I see the doc pages list this as 0.4.0.  Is there an updated page for this?  Is that page still valid?

**Comment by eeintech** [comment, marked as answer]
HI @lookme2 Yes the doc version is `0.4.0` because this is the last tagged release of the doc repository but what's on the website is the doc for the latest version of InvenTree by default. You can switch to another version of InvenTree using the little tool in the bottom right corner:

![image](https://user-images.githubusercontent.com/4020546/158655983-0da9014a-aef0-40c4-9ff2-020bbb424396.png)

**Reply by lookme2** [reply]
Thanks!
